File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/SimpleDependency_ViewModel.cs
/// <filename>
///     SimpleDependency_ViewModel.cs
/// </filename>
namespace dependencies
{
    public class SimpleDependency_ViewModel
    {
        private bool MyBool;

        public virtual bool getMyBool()
        {
            return this.MyBool;
        }

        public virtual void loadViewModel(string Path)
        {
        }
    }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/SimpleDependency_ViewModel.cpp
#include "SimpleDependency_ViewModel.h"
#include <string>

namespace dependencies
{
  bool SimpleDependency_ViewModel::getMyBool()
  {
    return this->MyBool;
  }
  void SimpleDependency_ViewModel::loadViewModel(std::string Path)
  {
  }
}
